### ISSUE TITLE
:comet: Epic: Add support from AFlex to Hyrax for dynamic views, for m3 profiles

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/amigos#11]
[SOW](https://github.com/scientist-softserv/amigos/issues/1)
[Spike
](https://github.com/scientist-softserv/amigos/issues/10)
# Summary

In Allinson Flex, the work show page reflects the imported schemas (defined by the uploaded yamls). 

Hyrax needs to be dynamic in this way. 

## Tasks

- [x] #30
- [x] #31
- [x] Future? How can we incorporate internalization for dynamic labels?
- [x] #32

## Nice to Have
- [ ] #27
- [ ] #33
- [ ] #34

# Acceptance Criteria

- [ ] The work's view is defined by the uploaded metadata schema files. 

# Screenshots or Video

# Testing Instructions

# Notes

**Comment by ShanaLMoore** [comment]
Estimating as a 5. May change after #10

**Comment by ShanaLMoore** [comment]
Moving to Nice to Haves because this matches feature parity with Allinson Flex as is. 

- [ ] #33
- [ ] #34
